pandas/core/arrays/interval.py

```diff
@@ -195,13 +195,13 @@ def _simple_new(
         left = ensure_index(left, copy=copy)
         right = ensure_index(right, copy=copy)
 
-        if dtype is not None:
+        if dtype:
```

Comment:
This doesn't seems right, are you sure about this (I could be 

(Also appears in other spots)

Reply:
Apologizes, there is a ```not``` in there, my bad.

Reply:
Yep, that is why. I will probably hunt those in the mater repo because with some benchmarks I did I found there is a hair of performance improvement when you remove the `is not None`

Reply:
can you revert the last commit. and do this in a separate PR.

Reply:
Sure